src/components/Hero/HomeHero/index.tsx

```diff
@@ -33,11 +33,11 @@ const HomeHero = ({ heroImg }: HomeHeroProps) => {
           maxW="2xl"
         >
           <Morpher />
-          <VStack spacing="6">
+          <VStack spacing="6" maxW={{ lg: "2xl" }}>
             <Heading as="h1" size="2xl">
               {t("page-index:page-index-title")}
             </Heading>
-            <Text size="xl">{t("page-index:page-index-description")}</Text>
```

Comment:
Is this correct? I see we are using `xl` in the DS.
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/afddbb94-aa57-4ec1-9491-7539a05101fd)

Reply:
Whoops! I needed to change the `size` instead of removing it.

This has been fixed.